epub33/fxl-a11y-tech/index.html

```diff
@@ -315,6 +342,921 @@ <h3>Legibility</h3>
+                        <pre>
+                            <code>
+                            &lt;meta property="schema:accessMode"&gt;auditory&lt;/meta&gt;
+                            </code>
+                            In addition, since there is a textual figcaption description present the additional metadata would be also included:
+                            <code>
+                            &lt;meta property="schema:accessMode"&gt;textual&lt;/meta&gt;
+                            </code>
+                        </pre>
+                        <div class="note">
+                            <p>Ideally there would also be a textual transcript included, and depending on its existance this would affect the accessModeSufficient metadata inclusion of 'textual'.</p>
+                        </div>
+                    </aside>
+                </section>
+
+                <section id="accessmode-text-on-visual">
+                    <h4>AccessMode: Text On Visual</h4>
+                    <p>Indicates that the resource contains text encoded in visual form.</p>
+                    <aside id="example-access-mode-text-on-visual" class="example" title="AccessMode: Text On Visual">
+                        <pre>
+                            <code>
+                            &lt;p&gt;
+                                &lt;img src="images/newspaper-clipping.jpg" alt="Newspaper Clipping: Climate Crisis: Local Scientist Speaks Out. By Jane Wilson. Our measurements indicate a 2.3°C increase over the past decade alone."&gt;
+                            &lt;/p&gt;
+                            </code>
+                        </pre>
+                       <p>The metadata to included in the OPF file would be:</p>
+                        <pre>
+                            <code>
+                            &lt;meta property="schema:accessMode"&gt;textOnVisual&lt;/meta&gt;
+                            </code>
+                            In addition, since there is an image and associated alt text description of the text on the newspaper cliplling the additional metadata would be also included:
+                            <code>
+                            &lt;meta property="schema:accessMode"&gt;visual&lt;/meta&gt;
+                            &lt;meta property="schema:accessMode"&gt;textual&lt;/meta&gt;
+                            &lt;meta property="schema:accessibilityFeature"&gt;alternativeText&lt;/meta&gt;
+                            </code>
+                        </pre>
+                    </aside>
+                </section>
+            </section>
+
+            <section id="metadata-access-mode-sufficient">
+                <h3>Access Mode Sufficient</h3>
+                <section>
+                    <h4>Single Values</h4>
+                    
+                    <p>Having a single accessMode implys that the entire content within the EPUB can be consumed with only that mode of accessing the content within the book.</p>
```

Comment:
accessMode or accessModeSufficient? This sentence is a little confusing and these properties are already really confusing!